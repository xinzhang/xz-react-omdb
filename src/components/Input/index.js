import styled from 'styled-components';

const Input = styled.input`
  width: 93%;
  height: 20px;
  padding: 8px;
  border-radius: 8px;
  border: 2px solid #cdcdcd;
  border-color: rgba(0, 0, 0, 0.14);
  background-color: AliceBlue;
  font-size: 14px;
`;

export default Input;
