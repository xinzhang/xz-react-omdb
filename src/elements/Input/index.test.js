import React from 'react';
import renderer from 'react-test-renderer';
import Input from '.';

describe('<FieldItem /> Component', () => {
  test('it should match the FieldItem with value', () => {
    const component = renderer.create(
      <Input placeholder="abc" value="test" onChange={() => {}} />,
    );
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
