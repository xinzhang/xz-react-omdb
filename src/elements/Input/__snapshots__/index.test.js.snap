// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<FieldItem /> Component it should match the FieldItem with value 1`] = `
<input
  className="sc-AxjAm cUzdnk"
  onChange={[Function]}
  placeholder="abc"
  value="test"
/>
`;
